benchmarks_install.py

```diff
@@ -7,77 +8,57 @@
 changes in ndindex were made in tandem with corresponding commits in
 versioned-hdf5.
 """
-
-import builtins
-import sys
-import os
+import logging
 import subprocess
+import sys
 
-# The first commit in versioned-hdf5 that is not compatible with ndindex 1.5
-ndindex_16_commit = 'af9ba2313c73cf00c10f490407956ed3c0e6467e'
-
-def print(*args):
-    # If we don't flush stdout, print output is out of order with run()
-    # output in the asv run -v log.
-    builtins.print(*args)
-    sys.stdout.flush()
+logger = logging.getLogger(__name__)
 
-def run(command, *args, **kwargs):
-    print(' '.join(command))
-    kwargs.setdefault('check', True)
-    return subprocess.run(command, *args, **kwargs)
+# The first commit in versioned-hdf5 that is not compatible with ndindex 1.5;
+# this was released in versioned-hdf5 1.2.2
+ndindex_1_5_pin_version = '1.2.2'
```

Comment:
ndindex 1.6 is almost 3 years old; versioned_hdf5 is 1.2.2 is almost 4 years old (which makes no sense to me given the comment above). 

Does anyone even care about versions this old? Shouldn't we just stat from a sane minimum version e.g. whatever was published in the last 2 years?

IMHO I would delete this whole script.

Reply:
Sure, I'm happy to delete this entirely.